documentation/how-to/custom-domain.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Configure a Custom Domain
+---
+
+<!-- overview -->
+
+This guide explains how to configure a custom domain for your Gateway Service(s).
+
+<!-- prerequisites -->
+
+## Before you begin
+
+Before you begin, ensure you:
+
+- [Install gwa CLI](/how-to/gwa-install.md)
+- [Create a Gateway](/how-to/create-gateway.md)
+
+<!-- steps -->
+
+## Create a Gateway Service with a Custom Domain
+
+If you have already created a Gateway Service in [Create a Gateway Service](/how-to/create-gateway-service.md) 
+or [Quick Start](/tutorials/quick-start.md) using the `gwa generate-config` command, you will need to extract 
+the Gateway Service information into a separate YAML file and reformat as shown below.
+
+Example configuration:
+
+```yaml
+certificates:
+  - cert: "<PEM FORMAT>"
+    key: "<PEM FORMAT>"
+    tags: [ns.<gatewayId>]
+    id: <GENERATED UUID4>
+    snis:
+      - name: <HOSTNAME>
+        id: <GENERATED UUID4>
+        tags: [ns.<gatewayId>]
+services:
+  - name: example-service-name
+    host: httpbin.org
+    tags: [ns.<gatewayId>]
+    port: 443
+    protocol: https
+    routes:
+      - name: example-route-name
+        tags: [ns.<gatewayId>]
+        hosts:
+          - <HOSTNAME>
+```
+
+Where:
+
+- `certificates[n].id` is a UUID4 you will generate.
+- `certificates[n].snis.name` is the hostname on the custom domain
+- `certificates[n].snis.id` is a different UUID4 you will generate.
```

Comment:
```suggestion
- `certificates[n].snis[n].name` is the hostname on the custom domain
- `certificates[n].snis[n].id` is a different UUID4 you will generate.
```

since `snis` is also an array